test/server_test.py

```diff
@@ -1,17 +1,18 @@
-from pocha import describe, it, before, beforeEach, afterEach
-from hamcrest import *
-from unittest.mock import patch
-import scrypt
 from base64 import b64encode
+import json
 import re
+from unittest.mock import patch
 import yaml
+
+from hamcrest import *
 from peewee import fn
-import json
+from pocha import afterEach, before, beforeEach, describe, it
+import scrypt
 
-from server import app as server_app, limiter
-import util
 from db import User, Code, PendingEmail, LoginCode
+from server import app as server_app, limiter
```

Comment:
This actually looks confusing. Are we importing app as server_app and limiter as limiter? Or are we importing app under both the names server_app and limiter? Maybe brackets, line separation, or something?

Reply:
As much as I hate citing "pythonic" ways of doing things, this is just one of those Python things. There's not really a nicer way to do this, afaik.

Reply:
I know you can add parenthesis and have each import on it's own line, but I honestly don't like seeing that.